apps/aot/src/app/events/[year]/leaderboard/_components/overall-leaderboard.tsx

```diff
@@ -23,7 +56,7 @@ async function getOverallLeaderboard(year: number, isPast: boolean) {
   SELECT
     u.name,
```

Comment:
```suggestion
  SELECT
    u.id as userId
    u.name,
```

This will help with user data lookup, using the primary key.

Reply:
Also make sure to update: `OverallLeaderboardEntry` to include `userId`!